Makefile

```diff
@@ -4,6 +4,8 @@ IMAGE_TAG ?= devel
 # Image URL to use all building/pushing image targets
 IMG ?= "${IMAGE_REGISTRY}/${IMAGE_NAME}:${IMAGE_TAG}"
 
+OCI_BIN ?= podman
```

Comment:
`OCI_BIN ?= $(shell if podman ps >/dev/null 2>&1; then echo podman; elif docker ps >/dev/null 2>&1; then echo docker; fi)` can be used to fins the oci bin.